backend/routes/leaderboardRoutes.js

```diff
@@ -1,18 +1,20 @@
-const express = require("express");
```

Comment:
Use import if:

You're writing modern JavaScript and want to leverage ES Modules.
Your project is configured to use ES Modules (e.g., .mjs or "type": "module" in package.json).

Reply:
there were both import and require.